src/components/common/RegistriesContactInfo.vue

```diff
@@ -22,22 +22,9 @@
       </v-icon>
       <span class="px-2">Victoria Office:</span>
       <a
-        href="tel:+1-250-387-7848"
+        href="tel:+1-250-370-1033"
```

Comment:
change the help number and remove email
![image](https://github.com/bcgov/business-create-ui/assets/144159934/be57c3dd-2d47-4818-b570-6e33e2b4dd17)

Reply:
This is an obsolete conversation -- there is now a new component for firm contact info. Resolving.